articles/virtual-machines/troubleshooting/error-messages.md

```diff
@@ -212,4 +212,4 @@ This section lists the common error messages you may encounter when managing VMs
 
 
 ## Next steps
-If you need more help, you can contact the Azure experts on [the MSDN Azure and Stack Overflow forums](https://azure.microsoft.com/support/forums/). Alternatively, you can file an Azure support incident. Go to the [Azure support site](https://azure.microsoft.com/support/options/) and select **Get Support**.
\ No newline at end of file
+If you need more help, you can contact the Azure experts on [the MSDN Azure and Stack Overflow forums](https://azure.microsoft.com/support/forums/). Alternatively, you can file an Azure support incident. Go to the [Azure support site](https://azure.microsoft.com/support/options/) and select **Get Support**.
```

Comment:
I am not sure what changed for this line. I didn't change it, I guess this change were caused by github editor.